components/prism-opencl.js

```diff
@@ -0,0 +1,49 @@
NT|CL_KERNEL_MAX_NUM_SUB_GROUPS|CL_KERNEL_MAX_SUB_GROUP_SIZE_FOR_NDRANGE|CL_KERNEL_NUM_ARGS|CL_KERNEL_PREFERRED_WORK_GROUP_SIZE_MULTIPLE|CL_KERNEL_PRIVATE_MEM_SIZE|CL_KERNEL_PROGRAM|CL_KERNEL_REFERENCE_COUNT|CL_KERNEL_SUB_GROUP_COUNT_FOR_NDRANGE|CL_KERNEL_WORK_GROUP_SIZE|CL_LINKER_NOT_AVAILABLE|CL_LINK_PROGRAM_FAILURE|CL_LOCAL|CL_LUMINANCE|CL_MAP_FAILURE|CL_MAP_READ|CL_MAP_WRITE|CL_MAP_WRITE_INVALIDATE_REGION|CL_MEM_ALLOC_HOST_PTR|CL_MEM_ASSOCIATED_MEMOBJECT|CL_MEM_CONTEXT|CL_MEM_COPY_HOST_PTR|CL_MEM_COPY_OVERLAP|CL_MEM_FLAGS|CL_MEM_HOST_NO_ACCESS|CL_MEM_HOST_PTR|CL_MEM_HOST_READ_ONLY|CL_MEM_HOST_WRITE_ONLY|CL_MEM_KERNEL_READ_AND_WRITE|CL_MEM_MAP_COUNT|CL_MEM_OBJECT_ALLOCATION_FAILURE|CL_MEM_OBJECT_BUFFER|CL_MEM_OBJECT_IMAGE1D|CL_MEM_OBJECT_IMAGE1D_ARRAY|CL_MEM_OBJECT_IMAGE1D_BUFFER|CL_MEM_OBJECT_IMAGE2D|CL_MEM_OBJECT_IMAGE2D_ARRAY|CL_MEM_OBJECT_IMAGE3D|CL_MEM_OBJECT_PIPE|CL_MEM_OFFSET|CL_MEM_READ_ONLY|CL_MEM_READ_WRITE|CL_MEM_REFERENCE_COUNT|CL_MEM_SIZE|CL_MEM_SVM_ATOMICS|CL_MEM_SVM_FINE_GRAIN_BUFFER|CL_MEM_TYPE|CL_MEM_USES_SVM_POINTER|CL_MEM_USE_HOST_PTR|CL_MEM_WRITE_ONLY|CL_MIGRATE_MEM_OBJECT_CONTENT_UNDEFINED|CL_MIGRATE_MEM_OBJECT_HOST|CL_MISALIGNED_SUB_BUFFER_OFFSET|CL_NONE|CL_NON_BLOCKING|CL_OUT_OF_HOST_MEMORY|CL_OUT_OF_RESOURCES|CL_PIPE_MAX_PACKETS|CL_PIPE_PACKET_SIZE|CL_PLATFORM_EXTENSIONS|CL_PLATFORM_HOST_TIMER_RESOLUTION|CL_PLATFORM_NAME|CL_PLATFORM_PROFILE|CL_PLATFORM_VENDOR|CL_PLATFORM_VERSION|CL_PROFILING_COMMAND_COMPLETE|CL_PROFILING_COMMAND_END|CL_PROFILING_COMMAND_QUEUED|CL_PROFILING_COMMAND_START|CL_PROFILING_COMMAND_SUBMIT|CL_PROFILING_INFO_NOT_AVAILABLE|CL_PROGRAM_BINARIES|CL_PROGRAM_BINARY_SIZES|CL_PROGRAM_BINARY_TYPE|CL_PROGRAM_BINARY_TYPE_COMPILED_OBJECT|CL_PROGRAM_BINARY_TYPE_EXECUTABLE|CL_PROGRAM_BINARY_TYPE_LIBRARY|CL_PROGRAM_BINARY_TYPE_NONE|CL_PROGRAM_BUILD_GLOBAL_VARIABLE_TOTAL_SIZE|CL_PROGRAM_BUILD_LOG|CL_PROGRAM_BUILD_OPTIONS|CL_PROGRAM_BUILD_STATUS|CL_PROGRAM_CONTEXT|CL_PROGRAM_DEVICES|CL_PROGRAM_IL|CL_PROGRAM_KERNEL_NAMES|CL_PROGRAM_NUM_DEVICES|CL_PROGRAM_NUM_KERNELS|CL_PROGRAM_REFERENCE_COUNT|CL_PROGRAM_SOURCE|CL_QUEUED|CL_QUEUE_CONTEXT|CL_QUEUE_DEVICE|CL_QUEUE_DEVICE_DEFAULT|CL_QUEUE_ON_DEVICE|CL_QUEUE_ON_DEVICE_DEFAULT|CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE|CL_QUEUE_PROFILING_ENABLE|CL_QUEUE_PROPERTIES|CL_QUEUE_REFERENCE_COUNT|CL_QUEUE_SIZE|CL_R|CL_RA|CL_READ_ONLY_CACHE|CL_READ_WRITE_CACHE|CL_RG|CL_RGB|CL_RGBA|CL_RGBx|CL_RGx|CL_RUNNING|CL_Rx|CL_SAMPLER_ADDRESSING_MODE|CL_SAMPLER_CONTEXT|CL_SAMPLER_FILTER_MODE|CL_SAMPLER_LOD_MAX|CL_SAMPLER_LOD_MIN|CL_SAMPLER_MIP_FILTER_MODE|CL_SAMPLER_NORMALIZED_COORDS|CL_SAMPLER_REFERENCE_COUNT|CL_SIGNED_INT16|CL_SIGNED_INT32|CL_SIGNED_INT8|CL_SNORM_INT16|CL_SNORM_INT8|CL_SUBMITTED|CL_SUCCESS|CL_UNORM_INT16|CL_UNORM_INT24|CL_UNORM_INT8|CL_UNORM_INT_101010|CL_UNORM_INT_101010_2|CL_UNORM_SHORT_555|CL_UNORM_SHORT_565|CL_UNSIGNED_INT16|CL_UNSIGNED_INT32|CL_UNSIGNED_INT8|CL_VERSION_1_0|CL_VERSION_1_1|CL_VERSION_1_2|CL_VERSION_2_0|CL_VERSION_2_1|CL_sBGRA|CL_sRGB|CL_sRGBA|CL_sRGBx)\b/,
```

Comment:
Same comment here. They seem to all share the `CL_` prefix, plus other parts are ofter repeated.